com.neatdecisions.Detwinner.json

```diff
@@ -0,0 +1,180 @@
+{
+  "app-id": "com.neatdecisions.Detwinner",
+  "runtime": "org.gnome.Sdk",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "detwinner",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=host",
+    "--filesystem=xdg-config/gtk-3.0",
+    "--talk-name=org.gtk.vfs",
+    "--talk-name=org.gtk.vfs.*",
+    "--talk-name=org.freedesktop.portal.Trash"
+  ],
+  "build-options": {
```

Comment:
these options are included by default in this version of the SDK and it's redundant here.